cli/cli.py

```diff
@@ -79,9 +79,10 @@ def bestbuy(sku):
 
 
 @click.command()
-def evga():
+@click.option('--test', is_flag=True)
+def evga(test):
     ev = Evga()
-    ev.buy()
+    ev.buy(test)
```

Comment:
this should be

`ev.buy(test=test)`

Its passing test as the delay parameter here

Reply:
Fixed.